app/session.server.ts

```diff
@@ -82,7 +82,7 @@ export async function getAuthenticatedUser(
   const sessionCookie = await getUserSession(request);
   if (sessionCookie) {
     const validCookie = await auth.verifySessionCookie(sessionCookie);
-    return getUserById(validCookie.uid);
+    return getUserById(validCookie.id);
```

Comment:
This is a firebase function as well so it needs to be uid.

```suggestion
    return getUserById(validCookie.uid);
```

I wonder if it may be best to switch back to uid for the database as well? That's a thought for another day though